mmv1/templates/terraform/constants/vertex_ai_model.go.tmpl

```diff
@@ -1,4 +1,8 @@
 func vertexAiModelDiffSuppress(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	// we require diffSuppress on COPY method and not UPLOAD
```

Comment:
this is included so that we can fill out the description field on create when wanting to upload.

As a reminder this diffsuppress was created because of copy not supporting description field on create. More info can be found in this thread. https://github.com/GoogleCloudPlatform/magic-modules/pull/12074#discussion_r1829907452